.cspell/custom-dictionary.txt

```diff
@@ -0,0 +1,90 @@
+# Custom Dictionary Words
```

Comment:
We could separate the words according to the content, e.g. docs, pynxtools/nexus something like this.